VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -923,7 +923,7 @@ export abstract class WaitingRoomBaseDirective {
         this.videoCallService.stopScreenShare();
     }
 
-    get shouldMuteHearing(): boolean {
+    shouldMuteHearing(): boolean {
```

Comment:
Reverting back to a regular method instead of getter to allow super to be called in components that extend this.